assets/js/barChart.js

```diff
@@ -149,8 +185,11 @@ class BarChart {
 
         // Transform license count into array of objects with license name & count
         let chartData = $.map(count, function(val, key) {
+            let trimName = "";
```

Comment:
Assigning to an empty string is unneeded. Instead do this:

`let trimName = key.substring(0, 50) + (key.length > 50 ? " ..." : "");`